appholder/src/main/java/com/android/identity/wallet/util/PreferencesHelper.kt

```diff
@@ -31,7 +31,12 @@ object PreferencesHelper {
         // As per the docs, the credential data contains reference to Keystore aliases so ensure
         // this is stored in a location where it's not automatically backed up and restored by
         // Android Backup as per https://developer.android.com/guide/topics/data/autobackup
-        return context.noBackupFilesDir
+
+        val storageDir = File(context.noBackupFilesDir, "appholder")
```

Comment:
It's because we're breaking the on-disk format. Without this change, the application will load old data and crash. With this change, the app won't crash but it'll no credentials. At this point, such a breaking change is fine (it's just a test app after all) but as the commit message says, in the future we'll commit to stable file formats.

Reply:
gotcha, thanks!